src/Symfony/Component/Notifier/Bridge/KazInfoTeh/KazInfoTehTransport.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\KazInfoTeh;
+
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Egor Taranov <dev@taranovegor.com>
+ */
+class KazInfoTehTransport extends AbstractTransport
+{
+    protected const HOST = 'kazinfoteh.org';
+
+    /**
+     * @var string
+     */
+    private $username;
+
+    /**
+     * @var string
+     */
+    private $password;
+
+    /**
+     * @var string
+     */
+    private $sender;
```

Comment:
Question for @symfony/mergers, do we want property type hints for new features? I guess yes

Reply:
I think property typehint were an issue for preloading (cf @jderusse issue)

Reply:
only for classes that are not autoladable IIRC, scalar types should not be affected